docs/assets/scss/main.scss

```diff
@@ -1,5 +1,9 @@
 @import '../../themes//docs/assets/scss/main.scss';
 
+:root.dark {
+  --color-text: #c2c2c2;
```

Comment:
I was feeling that the text was a bit hard to read due to missing contrast. I have checked the accessibility color, and it was only validating [AA minimum ratio](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Guides/Understanding_WCAG/Perceivable/Color_contrast) and not AAA. I have updated to a color that validates AAA ratio.